__mocks__/@azure/storage-blob.js

```diff
@@ -25,10 +25,9 @@ export class BlobServiceClient {
     return new Container()
   }
 
-  static fromConnectionString () {}
+  static fromConnectionString () {
+    return new BlobServiceClient()
+  }
 }
 
-BlobServiceClient.fromConnectionString = jest.fn()
-  .mockImplementation(() => {
-    return new BlobServiceClient()
-  })
+module.exports = { BlobServiceClient }
```

Comment:
Is it necessary to go back to commonJS style exports for this change?

Reply:
aah no this is just work I did before the rebasing. Will update.